code/modules/surgery/organs/ears.dm

```diff
@@ -106,11 +106,60 @@
 	damage_multiplier = 0.9
 	organ_flags = ORGAN_SYNTHETIC
 
-/obj/item/organ/internal/ears/cybernetic/upgraded
-	name = "upgraded cybernetic ears"
+/obj/item/organ/internal/ears/cybernetic/translation
+	name = "cybernetic translation ears"
 	icon_state = "ears-c-u"
-	desc = "An advanced cybernetic ear, surpassing the performance of organic ears."
+	desc = "Allows the user to understand one of several commonly spoken languages. Grants no ability to speak these languages."
 	damage_multiplier = 0.5
+	var/list/language_list = list(/datum/language/common, /datum/language/uncommon, /datum/language/moffic, /datum/language/draconic, /datum/language/calcic, /datum/language/voltaic, /datum/language/nekomimetic)
```

Comment:
This should probably check what species are actually roundstart available instead, Admins decide what species are playable.

Reply:
It could check for that if absolutely necessary, but none of the other relevant ways to get an extra language (bilingual quirk, translation radio keys) do so, so I'm wondering why it would be necessary to check that in this case.